openedx/adg/lms/applications/tests/test_helpers.py

```diff
@@ -324,3 +327,30 @@ def test_get_extra_context_for_application_review_page(mock_get_application_revi
     actual_context = get_extra_context_for_application_review_page(user_application)
 
     assert expected_context == actual_context
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'business_line_names, is_superuser, is_staff, expected_result',
+    [
+        ([], False, False, False),
+        ([], True, False, False),
+        (['business-line-1'], False, False, False),
+        (['business-line-1'], True, False, False),
+        ([], False, True, False),
+        ([], True, True, True),
+        (['business-line-1'], False, True, True),
+        (['business-line-1', 'business-line-2'], False, True, True),
+        (['business-line-1'], True, True, True),
+    ]
+)
+def test_is_user_business_line_admin(business_line_names, is_superuser, is_staff, expected_result):
+    """
+    Test if the user is a superuser or an ADG admin or the admin of any Business line while having the staff
+    status
+    """
+    business_line_iter = factory.Iterator(business_line_names)
+    business_lines = BusinessLineFactory.create_batch(len(business_line_names), title=business_line_iter)
+    test_user = UserFactory(groups=business_lines, is_superuser=is_superuser, is_staff=is_staff)
+
+    assert has_admin_permissions(test_user) == expected_result
```

Comment:
The test name indicates it is a tested for class method `is_user_business_line_admin ` but it is testing different method `has_admin_permissions' and indirectly testing `is_user_business_line_admin`


Please write seperate tests for `is_user_business_line_admin` and `has_admin_permissions` and mock external dependencies